resources/functionMap.php

```diff
@@ -327,15 +327,15 @@
 'ArrayAccess::offsetUnset' => ['void', 'offset'=>'mixed'],
 'ArrayIterator::__construct' => ['void', 'array='=>'array|object', 'flags='=>'int'],
 'ArrayIterator::append' => ['void', 'value'=>'mixed'],
-'ArrayIterator::asort' => ['void'],
+'ArrayIterator::asort' => ['true'],
```

Comment:
I worry that with this change, these methods lose "always have a side effect" information. You need to add these here https://github.com/phpstan/phpstan-src/blob/master/bin/functionMetadata_original.php, and run `php ./bin/generate-function-metadata.php`. And commit the result :)